webview-ui/src/components/ResolutionsPage/ModifiedFile/HunkSelectionInterface.tsx

```
@@ -0,0 +1,134 @@
+  onHunkStateChange,
+  actionTaken,
+  filePath,
+}) => {
+  const getHunkStatusBadge = (hunkId: string) => {
+    const state = hunkStates[hunkId];
+    switch (state) {
+      case 'accepted':
+        return <Badge isRead>Accepted</Badge>;
+      case 'rejected':
+        return <Badge isRead>Rejected</Badge>;
+      case 'pending':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)